code/modules/mob/living/carbon/human/life.dm

```diff
@@ -33,6 +33,10 @@
 	if(stat != DEAD)
 		//Stuff jammed in your limbs hurts
 		handle_embedded_objects()
+		
+	if(stat != DEAD)
+		//yogs - bandage memes
+		handle_bandaged_limbs()
```

Comment:
muh modular

Reply:
Same here as the other thing.  Unless you just want it tagged around?